draft-ietf-sedate-datetime-extended.md

```diff
@@ -176,15 +175,15 @@ UTC:
   International des Poids et Mesures (BIPM) in conjunction with leap
   seconds as announced by the International Earth Rotation and
   Reference Frames Service {{IERS}}.
-  From 1972 through 1987 UTC was maintained entirely by Bureau
+  From 1972 through 1987, UTC was maintained entirely by Bureau
   International de l'Heure (BIH).
-  Before 1972 UTC was not generally recognized and civil time was
+  Before 1972, UTC was not generally recognized and civil time was
   determined by individual jurisdictions using different techniques
   for attempting to follow Universal Time based on measuring the
   rotation of the earth.
 
   UTC is often mistakenly referred to as GMT, an earlier timescale
-  UTC was designed to be a useful successor for.
+  that UTC was designed to supersede.
```

Comment:
I don't think that UTC (an international time) can aspire to "supersede" GMT (a UK time).  I'd stick with the carefully chosen wording.

Reply:
As far as I can tell, "supercede" and "successor" are conceptually interchangeable (both referring to the temporal sequence in which distinct entities occupy the same position) and in fact "superseder" is synonymous with "successor" according to https://www.merriam-webster.com/thesaurus/superseder . But perhaps we can find an even better phrasing?
```suggestion
  that predates the introduction of Universal Time and currently
  coincides with a UTC offset of 00:00.
```

Reply:
I have a different dictionary, which does not equate "succeed" and "supersede" as much.
I really think that what the original text says is an accurate description, and trying to wordsmith this in a way that insinuates a relationship beyond the fact that one was inspiring the definition of the other is going to create difficulties.  I also don't want to open the whole can of worms of UT1 etc. here.  So I stand by the original text.

Reply:
BTW, https://en.wikipedia.org/wiki/Greenwich_Mean_Time has a nice description of the different ongoing meanings of GMT (mean solar time, civil time in the UK during winter), none of which is actually superseded by UTC.

Reply:
> ongoing meanings of GMT... none of which is actually superseded by UTC

Isn't that in itself an argument against keeping the current "UTC was designed to supersede" language?

Reply:
I don't know what "current" means here, but the pre-PR language is about UTC being "a useful successor for" GMT.  That is why I would like to keep it.